mealpy/mealpy.py

```diff
@@ -45,31 +45,34 @@ def login(self, user, password):
 
         return request.status_code
 
-    def get_cities(self):
-        request = self.session.post(CITIES_URL)
-        request.raise_for_status()
-        return request.json()['result']
+    @staticmethod
+    def get_cities():
+        response = requests.post(CITIES_URL)
```

Comment:
Minor nit, but it would be more consistent to have all the requests go through `self.session` as before even though it doesn't really have to.

Reply:
I disagree about the need for consistency. It would reuse headers and cookies unnecessarily (functional change), for the sake of consistency (code styling). This change deliberately moves away from that consistency and demonstrates that auth (#34) or state was never required.